source/code/source/playfab/QoS/XPlatSocket.cpp

```diff
@@ -166,12 +166,12 @@ namespace PlayFab
             FD_CLR(0, &socketSet);
             FD_SET(s, &socketSet);
 
-            auto fd_max = s+1;
+            int fd_max = s+1;
```

Comment:
>s [](start = 25, length = 1)

`s` is a different type depending on platform (look at XPlatSocket.h)

instead of worrying about the type it may be simpler to just embed this in the select call

`select(s + 1, ...);`